pkgs/stdenv/generic/setup.sh

```diff
@@ -1512,14 +1502,14 @@ showPhaseFooter() {
     local startTime="$2"
     local endTime="$3"
     local delta=$(( endTime - startTime ))
-    (( $delta < 30 )) && return
+    (( delta < 30 )) && return
```

Comment:
Is the $ unneeded in math expressions?

Reply:
Yeah 

SC2004 (style): $/${} is unnecessary on arithmetic variables.